openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1625,31 +1625,176 @@ TBD
+* Change controller: OpenID Foundation Digital Credentials Protocols Working Group - openid-specs-digital-credentials-protocols@lists.openid.net
+* Reference: (#credential-request-errors) of this specification
+
+* Name: unsupported_credential_format
+* Usage Location: resource access error response
+* Protocol Extension: OpenID for Verifiable Credentials Issuance
+* Change controller: OpenID Foundation Digital Credentials Protocols Working Group - openid-specs-digital-credentials-protocols@lists.openid.net
+* Reference: (#credential-request-errors) of this specification
+
+* Name: invalid_proof
+* Usage Location: resource access error response
+* Protocol Extension: OpenID for Verifiable Credentials Issuance
+* Change controller: OpenID Foundation Digital Credentials Protocols Working Group - openid-specs-digital-credentials-protocols@lists.openid.net
+* Reference: (#credential-request-errors) of this specification
+
+* Name: invalid_encryption_parameters
+* Usage Location: resource access error response
+* Protocol Extension: OpenID for Verifiable Credentials Issuance
+* Change controller: OpenID Foundation Digital Credentials Protocols Working Group - openid-specs-digital-credentials-protocols@lists.openid.net
+* Reference: (#credential-request-errors) of this specification
 
 ## Well-Known URI Registry
 
-This specification registers the well-known URI defined in (#credential-issuer-wellknown) in the IANA Well-Known URI registry defined in RFC 5785 [@!RFC5785].
+This specification registers the following well-known URI in the IANA "Well-Known URI" registry established by [@!RFC5785].
 
 * URI suffix: openid-credential-issuer
-* Change controller: AB/Connect Working Group - openid-specs-ab@lists.openid.net
+* Change controller: OpenID Foundation Digital Credentials Protocols Working Group - openid-specs-digital-credentials-protocols@lists.openid.net
 * Specification document: (#credential-issuer-wellknown) of this document
 * Related information: (none)
 
+## Media Types Registry
+
+This specification registers the following media types in the IANA "Media Types" registry [@!IANA.MediaTypes] in the manner described in [@!RFC6838].
+
+* Type name: `application`
+* Subtype name: `openid4vci-proof+jwt`
+* Required parameters: n/a
+* Optional parameters: n/a
+* Encoding considerations: Uses JWS Compact Serialization, as specified in [@!RFC7515].
+* Security considerations: See the Security Considerations in [@!RFC7519].
+* Interoperability considerations: n/a
+* Published specification: (#jwt-proof-type) of this specification
+* Applications that use this media type: Applications that issue and store verifiable credentials
+* Additional information:
+  - Magic number(s): n/a
+  - File extension(s): n/a
+  - Macintosh file type code(s): n/a
+* Person & email address to contact for further information: TBD
+* Intended usage: COMMON
+* Restrictions on usage: none
+* Author: Michael B. Jones, michael_b_jones@hotmail.com
```

Comment:
Why does this registration require an author? if so, it should be the draft authors, shouldn't it?

Reply:
An author is required because it's required in the registration template at https://www.rfc-editor.org/rfc/rfc6838.html#section-5.6.  The odd thing about this template is that the Author is separate from the Change Controller.  (For most IANA registries there's no Author - only a Change Controller.)

I used my name since I'm the author of the registration but I don't feel strongly about it staying that way.  One thing we could try is making the Author value the same as the Change Controller value and see if that flies.  What do you think?

Reply:
Agree with Torsten, this should be one of the spec authors

```suggestion
* Author: Torsten Lodderstedt, torsten@lodderstedt.net
```